### ISSUE TITLE
[Bug]: Document properties

### ISSUE THREAD

**Issue by martinthomson** [mozilla/pdf.js#19228]
### Attach (recommended) or Link to PDF file

https://zenodo.org/records/14107216

### Web browser and its version

Firefox Nightly 135.0a1 (2024-12-12) (64-bit)

### Operating system and its version

Windows

### PDF.js version

4.9.143

### Is the bug present in the latest PDF.js version?

Yes

### Is a browser extension

No

### Steps to reproduce the problem

Visit the page and select the "Document Properties..." option in the embedded document.

### What is the expected behavior?

A popup shows with document properties.

### What went wrong?

![image](https://github.com/user-attachments/assets/be399bae-06e1-4d12-bf5e-85dcd6b76ccf)

**Comment by Snuffleupagus** [comment]
TL;DR Please contact that website and ask them to update their outdated PDF.js version.

---

> ### Attach (recommended) or Link to PDF file
> 
> https://zenodo.org/records/14107216

Unfortunately that website uses a customized and *extremely outdated* version of the PDF.js library, hence it's *not* the Firefox built-in PDF Viewer you're seeing. According to the web console:
```
Warning: Setting up fake worker. [pdf.js:342:12](webpack://invenio-assets/node_modules/pdfjs-dist/build/pdf.js)
PDF 35e79f792768ac49b2210bfcab068d1d [1.7 - / Microsoft Word] (PDF.js: 1.10.100) [viewer.js:7347:14](webpack://invenio-assets/js/invenio_previewer/pdfjs/viewer.js)
```
Note that that version was released *over six years ago*, see https://github.com/mozilla/pdf.js/releases/tag/v1.10.100

> ### PDF.js version
> 
> 4.9.143
> ### Is the bug present in the latest PDF.js version?
> 
> Yes

Please note that the issue template asks you to check the latest PDF.js version, using https://github.com/mozilla/pdf.js#online-demo

> ### Steps to reproduce the problem
> 
> Visit the page and select the "Document Properties..." option in the embedded document.

WFM, when using https://mozilla.github.io/pdf.js/web/viewer.html
